test/jdk/java/lang/String/CompactString/MaxSizeUTF16String.java

```diff
@@ -128,15 +133,15 @@ public void testMaxUTF8_UTF16Encode() {
         final byte[] bytes1 = s.getBytes(StandardCharsets.UTF_8);
         assertEquals(3, bytes1.length, "UTF_8 encoded length of 0xffff");
 
-        int min = Integer.MAX_VALUE / bytes1.length - 1;
+        int min = Integer.MAX_VALUE / bytes1.length;
         int max = min + 3;
 
         // String of size min can be UTF_8 encoded.
         System.out.println("testing size: " + min);
         String s1 = s.repeat(min);
         byte[] bytes = s1.getBytes(StandardCharsets.UTF_8);
         int remaining = Integer.MAX_VALUE - bytes.length;
-        assertTrue(remaining >= bytes1.length, "remainder too large: " + remaining);
+        // assertTrue(remaining >= bytes1.length, "remainder too large: " + remaining);
```

Comment:
Did you consider, instead of modifying lines 136 and 144, just adding these two lines here?
```c
min += 1; // adjust for OpenJ9
max += 1;
```

Reply:
Good idea. Updated and re-tested via https://openj9-jenkins.osuosl.org/view/Test/job/Grinder/3473/